src/types/style.d.ts

```diff
@@ -0,0 +1,13 @@
+export interface styleType {
```

Comment:
```suggestion
export interface StyleProps {
```
라는 이름이 더 더울리는 것 같네요. `style`을 전달하기 위한 `props`이기 때문에,
그리고, interface의 이름을 명명할 떈 보통 대문자를 선행해서 사용합니다.
하지만 이 것 또한 보통 사용하는 것이지, 팀원간에 컨벤션을 설정하셨다면, 무관합니다.

```suggestion
/** 
@params {string} height - 높이에 대한 전달값 입니다.
@params {string} width - 너비에 대한 전달값 입니다.
...
*/
export interface StyleProps {
```
이런식으로 들어올 @params값을 확인할 수 있도록 주석을 달 수 있습니다.
자세한 내용은 `JSDOC`을 검색해 보세요
[ts에서 제공하는 jsdoc예시](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#type)

Reply:
캄사합니돠 ㅜㅜ